diff --git a/syntropy_sdk/api_client.py b/syntropy_sdk/api_client.py
index 892dfd0..35826ca 100644
--- a/syntropy_sdk/api_client.py
+++ b/syntropy_sdk/api_client.py
@@ -75,10 +75,6 @@ class ApiClient(object):
         # Set default User-Agent.
         self.user_agent = "Swagger-Codegen/0.1.0/python"
 
-    def __del__(self):
-        self.pool.close()
-        self.pool.join()
-
     @property
     def user_agent(self):
         """User agent for this API client"""
